website/blog/2020-04-10-cobalt.md

```diff
@@ -0,0 +1,369 @@
+---
+author: Tomasz Godzik
+title: Metals v0.8.4 - Cobalt
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We are happy to announce the release of Metals v0.8.4, which will most likely be
+the last 0.8.x version.
+
+<table>
```

Comment:
I can't remember exactly, but I think it's because Markdown requires column labels, we could do something like:

| Summary                    |     |
| -------------------------- | --- |
| Commits since last release | 174 |
| Merged PRs                 | 55  |
| Contributors               | 9   |
| Closed issues              | 13  |
| New features               | 3   |

but not:

| Commits since last release | 174 |
| Merged PRs                 | 55  |
| Contributors               | 9   |
| Closed issues              | 13  |
| New features               | 3   |

Reply:
ah, that makes sense, thanks